dask_cuda/benchmarks/local_cupy_map_overlap.py

```diff
@@ -34,191 +29,95 @@ def mean_filter(a, shape):
         return sp_convolve(a, a_k)
 
 
-async def _run(client, args):
+def bench_once(client, args, write_profile=None):
```

Comment:
I made the clusters and clients asynchronous=False, all the futures are immediately awaited on, so it didn't seem necessary to have async stuff littering the code.